tetris.py

```diff
@@ -177,9 +228,46 @@ def draw_board(canvas):
         x_gap += square_size
 
 
+def get_high_scores():
+    index = 0
+    treeview.delete(*treeview.get_children())
+    for high_score_dict in json_dict['high_scores']:
+        if index == 0:
+            tag = 'odd_row'
+        else:
+            tag = 'even_row'
+        treeview.insert(parent="", index="end", tags=tag, values=(high_score_dict['Time'], high_score_dict['Game Speed'], high_score_dict['Score']))
```

Comment:
The speed shows up in the treeview as "300". It's not very useful. Showing it as "300ms/block" would be much better.

Reply:
I think it would be better to just display a difficulty name instead of number value and make it a primary sorting tool

Reply:
I'll do this in another PR though.